tests/ui/privacy/privacy1.rs

```diff
@@ -1,25 +1,9 @@
+//@ add-core-stubs
 #![feature(lang_items, no_core)]
 #![no_core] // makes debugging this test *a lot* easier (during resolve)
 
-#[lang="sized"]
-pub trait Sized {}
-
-#[lang="copy"]
-pub trait Copy {}
-
-#[lang="deref"]
-pub trait Deref {
-    type Target;
-}
-
-#[lang="legacy_receiver"]
-pub trait LegacyReceiver: Deref {}
-
-impl<'a, T> Deref for &'a T {
-    type Target = T;
-}
-
-impl<'a, T> LegacyReceiver for &'a T {}
+extern crate minicore;
+use minicore::*;
```

Comment:
Problem [RESOLVE 1/2]: I think these privacy tests should likely *not* use `minicore` to simplify debugging resolve problems if this test ever fails. Bringing on `minicore` which has a *lot* of items can complicate the namespaces significantly.

Reply:
Will change back to non-minicore.